From: Daniel van Vugt <daniel.van.vugt@canonical.com>
Date: Fri, 25 Aug 2023 14:26:56 +0800
Subject: kms/crtc: Increase default deadline evasion to 800 microseconds

This seems to be enough to fix:
 * Constant stuttering on Apple Magic Trackpad 2 (90Hz)
 * Constant stuttering on Microsoft IntelliMouse Explorer 3.0 (125Hz)
 * Wake-from-idle frame skips on Microsoft Classic IntelliMouse (1000Hz)

Bug: https://gitlab.gnome.org/GNOME/mutter/-/issues/2974
Bug-Ubuntu: https://launchpad.net/bugs/2033719
Forwarded: yes
Last-Update: 2023-09-27
---
 src/backends/native/meta-kms-crtc.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/backends/native/meta-kms-crtc.c b/src/backends/native/meta-kms-crtc.c
index 932e5e6..b17e846 100644
--- a/src/backends/native/meta-kms-crtc.c
+++ b/src/backends/native/meta-kms-crtc.c
@@ -28,7 +28,7 @@
 #include "backends/native/meta-kms-update-private.h"
 #include "backends/native/meta-kms-utils.h"
 
-#define DEADLINE_EVASION_US 500
+#define DEADLINE_EVASION_US 800
 #define DEADLINE_EVASION_WITH_KMS_TOPIC_US 1000
 
 typedef struct _MetaKmsCrtcPropTable
